applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.h

```diff
@@ -23,6 +23,7 @@
 #include "compressible_potential_flow_application_variables.h"
 #include "utilities/geometry_utilities.h"
 #include "utilities/enrichment_utilities.h"
```

Comment:
Why do you need this one?

Reply:
In order to call https://github.com/KratosMultiphysics/Kratos/blob/47918b6540fdb9e58b5bf41729e273fb549fc214/applications/CompressiblePotentialFlowApplication/custom_elements/incompressible_potential_flow_element.cpp#L485-L487

Reply:
You need the enrichment or just a splitting with common shape functions?

Reply:
I need to split the element in two parts with same shape functions but different weight, is this what you mean with splitting with common shape functions?

Reply:
I mean that if you don't need the enrichment, the `EnrichmentUtilities` makes no sense anymore since there is the `ModifiedShapeFunctions`.

Reply:
I didn't know that ModifiedShapeFunctions existed, I will take a look at this. Thanks for the hint!

Reply:
@rubenzorrilla I substituted the EnrichmentUtilities by the ModifiedShapeFunctions and get slightly different results. Were you aware of this? Is it because a different approach is used to split the element?

I am being rather careful with this change because the solution is highly sensible to how the trailing edge element is split.

Reply:
It sounds a bit strange to me. The unique thing that comes to my mind is a tolerances issue. Actually I am not familiar with the `CalculateEnrichedShapeFuncions` since the, let's say "official", tool to compute the split shape functions is now the `ModifiedShapeFunctions`.

Maybe it is a good idea that we isolate one of those split elements in the trailing edge and we do a unit test with it and its level-set values.

Reply:
I did a unit test and for the test I get the same values with both methods. However in other cases I don't. I will look at it further and check whether it is something related to the tolerances as you say. I will let you know if I find something.

Reply:
@rubenzorrilla you were right! It is in fact a problem with the tolerances. Let me open a separate issue to post my problem.